search_index.rb

```diff
@@ -11,13 +11,26 @@
 
 require 'algoliasearch'
 require 'optparse'
-options = { skip_contributors: false }
-OptionParser.new do |opts|
-  opts.banner = "Usage: #{$0} [options]"
-  opts.on('--skip-contributors', 'Skip indexing contributor list') do |s|
-    options[:skip_contributors] = true
+
+def getContributors(name, id, election)
+  contrib = Committee.where(["\"Filer_ID\" = ?", id]).first
+    .calculation(:contribution_list)
+  return nil if contrib.nil?
+  return contrib.map do |contributor|
+    {
+      type: :contributor,
+      first_name: contributor['Tran_NamF'],
+      last_name: contributor['Tran_NamL'],
+      amount: contributor['Tran_Amt1'],
+      committee_name: name,
+      committee_id: id,
+      election_slug: election.name,
+      election_location: election.location,
+      election_date: election.date,
+      election_title: election.title,
+    }
   end
-end.parse!
+end
```

Comment:
couple ruby nitpicks, take them or leave them:

* Ruby function names typically use `snake_case`
* `contributors_to_committee` might be a better name
* You can use the ampersand operator like `&.calculation(:contributon_list)` to prevent the script from crashing if there is not a Committee with the given Filer_ID